src/components/search/popular-courses/PopularCoursesIndex.jsx

```diff
@@ -20,7 +20,7 @@ const PopularCoursesIndex = ({ title }) => {
     offerCatalogs,
   });
   const config = getConfig();
-  const contentType = title === COURSE_TITLE ? CONTENT_TYPE_COURSE : CONTENT_TYPE_PROGRAM;
+  const contentType = getContentTypeFromTitle(title);
```

Comment:
If we are reusing this same component for course/program/pathways then wouldn't it be better to rename it to something more generic?

Reply:
done